PIPReader/src/main/java/it/cnr/iit/ucs/pipreader/PIPReader.java

```diff
@@ -271,6 +273,23 @@ private final void setFilePath( String filePath ) {
         Reject.ifBlank( this.filePath );
     }
 
+    private void logOperation( String... strings ) {

```

Comment:
maybe we can hide these functions inside the journal?
logOperation
format
each component implementing a journal would have to replicate these functions